File: Jack.Model/DTOs/ProcessaPresenca.vb
Public Class ProcessaPresenca

    Public Overridable Property CodigoFamilia As Integer
    Public Overridable Property NomedaMae As String
    Public Overridable Property NumeroReunioes As Integer
    Public Overridable Property TotalReunioes As Integer
    Public Overridable Property PercentualReunioes As Double
    Public Overridable Property IsSacolinha As String
    Public Overridable ReadOnly Property Nivel As String
        Get
            Return "Nivel " + NumeroNivel.ToString()
        End Get
    End Property
    Public Overridable Property NumeroNivel As Integer

End Class

File: Jack.Model/DTOs/ProcessaPresenca.cs
namespace Jack.Model
{
    public class ProcessaPresenca
    {

        public virtual int CodigoFamilia { get; set; }
        public virtual string NomedaMae { get; set; }
        public virtual int NumeroReunioes { get; set; }
        public virtual int TotalReunioes { get; set; }
        public virtual double PercentualReunioes { get; set; }
        public virtual string IsSacolinha { get; set; }

        public virtual string Nivel
        {
            get { return "Nivel " + NumeroNivel.ToString(); }
        }

        public virtual int NumeroNivel { get; set; }

    }

}
